src/value.cpp

```diff
@@ -718,6 +736,25 @@ MemRef MemRef::subview(const vector<Expr> &offsets,
   }
 }
 
+Expr MemRef::conv(const MemRef &input,
+    const MemRef &filter,
+    const std::vector<smt::Expr> strides,
+    const std::vector<smt::Expr> dilations) {
+  auto [indices, expr] = input.ShapedValue::conv(filter, strides, dilations);
+
+  // we splat results into 1D memory layout
+  auto idx = Index::var("outputIdx", VarType::BOUND);
+  auto outputIndices = layout.getInverseIndices(idx);
+  auto outputExpr = expr.substitute(indices, outputIndices);
+  auto outputArray = Expr::mkLambda(idx, outputExpr);
+
+  // store output memref
+  auto success = isInBounds() & input.isInBounds() & filter.isInBounds() &
+    noalias(input) & noalias(filter) & storeArray(outputArray, Index::zero(), get1DSize());
```

Comment:
Check whether "output", "input", "filter" MemRef is inbounds.
Also I added noalias condition for output MemRef